content/guides/component-testing/quickstart-react.md

```diff
@@ -44,69 +44,47 @@ npx cypress open
 ## Configuring Component Testing
 
 Whenever you run Cypress for the first time in a project, the app will prompt
-you to set up either E2E testing or Component Testing. Choose Component Testing
-and step through the configuration wizard.
+you to set up either **E2E Testing** or **Component Testing**. Choose
+**Component Testing** and step through the configuration wizard.
 
-The Cypress Launchpad will detect your framework and generate all the necessary
-configuration files, and ensure all required dependencies are installed.
+<DocsImage 
+  src="/img/guides/component-testing/select-test-type.png" 
+  caption="Choose Component Testing"> </DocsImage>
 
-<!-- TODO: while it's nice to have this photo, it'd be even better to have a video! -->
+The Project Setup screen automatically detects your framework and bundler, which
+is React and Vite in our case. Cypress Component Testing uses your existing
+development server config to render components, helping ensure your components
+act and display in testing the same as they do in production.
 
-<img src="/img/component-testing-automatic-configuration.png" style="border: none; box-shadow: none; margin-bottom: 1rem;" />
-<p style="font-size: 0.85rem; text-align: center;">The Cypress Launchpad will scaffold all of these files for you.</p>
+<DocsImage 
+  src="/img/guides/component-testing/project-setup-react.png" 
+  caption=""> </DocsImage>
 
-After setting up component testing, you will be at the Browser Selection screen.
-
-<!-- TODO: Browser Selection screen with lots of browsers -->
-
-<!-- Extraneous, belongs somewhere else: This page displays all of the browsers on your computer that we support. This generally means you'll see all Chromium-based browsers, Electron, and Firefox that are on your computer.  -->
+Next, the Cypress setup will detect your framework and generate all the
+necessary configuration files, and ensure all required dependencies are
+installed.
 
-Pick your favorite and click "Launch" to open the Cypress app.
+<DocsImage 
+  src="/img/guides/component-testing/scaffolded-files.png" 
+  caption="The Cypress launchpad will scaffold all of these files for you">
+</DocsImage>
 
-<!-- Leaving out until we have a spec generator -->
-
-<!-- ### In an empty project
+After setting up component testing, you will be at the Browser Selection screen.
 
-In an empty project, we'll prompt you to use one of our new spec generators to
-create a file that matches your project's `specPattern`. -->
+Pick your favorite and click the "Start Component Testing" button to open the
```

Comment:
'Pick your favorite' makes it sound like its a game or something 😄 

...maybe "Pick your browser of choice" or even "Choose your browser"?